### ISSUE TITLE
fix litmus OSS tests

### ISSUE THREAD

**Issue by gshif** [influxdata/influxdb#19240]
There are 5 tests that started failing recently. Before the change the bucket name could have quotation marks in its name. After the change bucket names may not include quotation marks. Below are tests that need to get updated.
```
test_create_buckets_special_chars[DoubleQuotes\\"-] - src.cloud.rest_api.buckets.test_gateway_create_bucket.TestCreateBucketsAPI
src/cloud/rest_api/buckets/test_gateway_create_bucket.py
AssertionError: Actual '400' 'status code' does not equal to expected '201' 'status code' assert 400 == 201   +400   -201
test_delete_buckets_special_chars[DoubleQuotes\\"-] - src.cloud.rest_api.buckets.test_gateway_delete_bucket.TestDeleteBucketsAPI
src/cloud/rest_api/buckets/test_gateway_delete_bucket.py
AssertionError: Actual '400' 'status code' does not equal to expected '201' 'status code' assert 400 == 201   +400   -201
test_get_buckets_special_chars[DoubleQuotes\\"-] - src.cloud.rest_api.buckets.test_gateway_get_bucket.TestGetBucketsAPI
src/cloud/rest_api/buckets/test_gateway_get_bucket.py
AssertionError: Actual '400' 'status code' does not equal to expected '201' 'status code' assert 400 == 201   +400   -201
Less
test_update_buckets_name_single_char_nonalphanumeric_case[_-] - src.cloud.rest_api.buckets.test_gateway_update_buckets.TestUpdateBucketsNameAPI
src/cloud/rest_api/buckets/test_gateway_update_buckets.py
AssertionError: Actual '400' 'create bucket status code' does not equal to expected '422' 'create bucket status code' assert 400 == 422   +400   -422
test_update_buckets_name_special_chars[DoubleQuotes\\"-] - src.cloud.rest_api.buckets.test_gateway_update_buckets.TestUpdateBucketsNameAPI
src/cloud/rest_api/buckets/test_gateway_update_buckets.py
AssertionError: Actual '400' 'status code' does not equal to expected '201' 'status code' assert 400 == 201   +400   -201
```

From log files:
```
020-08-06 12:30:37,106-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-<--------- test_create_buckets_special_chars DoubleQuotes\"  START --------->
2020-08-06 12:30:37,106-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-#############################################################################
2020-08-06 12:30:37,106-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-test_create_buckets_special_chars DoubleQuotes\" STEP 1: Create Bucket "DoubleQuotes\""
2020-08-06 12:30:37,107-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-buckets_util.create_bucket() function is being called
2020-08-06 12:30:37,107-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO------------------------------------------------------

2020-08-06 12:30:37,107-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-buckets_util.create_bucket() :Creating Bucket with 'DoubleQuotes\"' name, '3600' retention rules and '39a56dbd7b32546a' org id
2020-08-06 12:30:37,107-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-RestLib.post() is called with parameters:
	base_url=http://localhost:9999
	path=/api/v2/buckets
	json=None
	auth=None
	data={"name":"DoubleQuotes\"", "retentionRules": [{"type":"expire","everySeconds":3600}], "orgID": "39a56dbd7b32546a"}
	headers={'Authorization': 'Token d14Dcu6Aeexd2T0_i4OSwxDnipigkyFp2oBDRCs4hG9aoVAiM4UXjSzmfpy5Hwshmd4qe82MPNj-2GDjuwJG3A=='}
	timeout=None
	params=None
	verify=False
2020-08-06 12:30:37,109-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-RestLib.post() - response code = 400
2020-08-06 12:30:37,109-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-RestLib.post() - response headers = {'Content-Type': 'application/json; charset=utf-8', 'X-Platform-Error-Code': 'invalid', 'Date': 'Thu, 06 Aug 2020 12:30:37 GMT', 'Content-Length': '121'}
2020-08-06 12:30:37,109-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-RestLib.post() - response url = http://localhost:9999/api/v2/buckets
2020-08-06 12:30:37,109-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-RestLib.post() - response text = {
	"code": "invalid",
	"message": "bucket name DoubleQuotes\" is invalid. Bucket names may not include quotation marks"
}
2020-08-06 12:30:37,109-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-buckets_util.create_bucket() REQUESTED_BUCKET_ID AND REQUESTED_BUCKET_NAME ARE NONE
2020-08-06 12:30:37,109-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-buckets_util.create_bucket() function returned : status = '400', bucket_id = 'None', bucket_name = 'None', org_id = 'None', rp = '0', error_message = '{'code': 'invalid', 'message': 'bucket name DoubleQuotes" is invalid. Bucket names may not include quotation marks'}'
2020-08-06 12:30:37,110-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-buckets_util.create_bucket() : status = '400'
error_message_headers = 'None'
e_msg = 'bucket name DoubleQuotes" is invalid. Bucket names may not include quotation marks'
e_code = 'invalid'
e_op = ''
e_err = ''
error_message = '{'code': 'invalid', 'message': 'bucket name DoubleQuotes" is invalid. Bucket names may not include quotation marks'}'

2020-08-06 12:30:37,110-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-buckets_util.create_bucket() function is done
2020-08-06 12:30:37,110-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-
2020-08-06 12:30:37,110-src.cloud.rest_api.buckets.test_gateway_create_bucket-INFO-Actual '400' 'status code' does not equal to expected '201' 'status code'
assert 400 == 201
  +400
  -201
```

**Comment by gshif** [comment]
This PR has a fix: https://github.com/influxdata/idpe/pull/8251. Once it is merged into master I will monitor tests running in OSS.
